landing-strip/core/src/main/java/com/novoda/landingstrip/TabsPagerListener.java

```diff
@@ -0,0 +1,85 @@
+package com.novoda.landingstrip;
+
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.view.ViewGroup;
+
+import com.novoda.landingstrip.LandingStrip.TabSetterUpper;
+
+public class TabsPagerListener implements OnPagerAdapterChangedListener {
```

Comment:
what is this class supposed to do?

The name is ambiguous, and the closest thing I'd associate with it from name is a ViewPager.OnPageChangeListener.